extra/alacritty-msg.man

```diff
@@ -14,6 +14,16 @@ Path for IPC socket creation
 .TP
 \fBcreate-window\fR
 Create a new window in the same Alacritty process
+.SH "CREATE-WINDOW OPTIONS"
+.TP
+\fB\-\-hold\fR
+Remain open after child process exits
+.TP
+\fB\-\-working\-directory\fR <working\-directory>
+Start the shell in the specified working directory
+.TP
+\fB\-e\fR, \fB\-\-command\fR <command>...
+Command and args to execute (must be last argument)
```

Comment:
Currently this is indented on the toplevel, how do you feel about indenting this below the `create-window` message and just calling it `OPTIONS`.

Should make it a little easier to understand without having to cross-reference the title.

Also the `SYNOPSIS` might need to be changed to document that after `[MESSAGES]` there's also another `[OPTIONS]`, or maybe `[MESSAGE_OPTIONS]`. Also I'd say `[MESSAGES]` should probably be `[MESSAGE]` because there can only be one.